### ISSUE TITLE
Enable supportsPanAndZoom

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/natural-selection#219]
@pixelzoom on latest dev (1.2.0-dev3) pinch to zoom did not seem to be enabled. I thought it was now on by default in master, is there currently some override or?

**Comment by pixelzoom** [comment]
There is no specific override in Natural Selection. @jessegreenberg?

**Comment by amanda-phet** [comment]
9/3/20 design meeting: not necessary for 10/1 client version, but we'd like this in the RC version.

**Comment by pixelzoom** [comment]
@ariel-phet said that we should opt-out for the 10/1 dev release.

**Comment by pixelzoom** [comment]
The general issue for enabling zoom by default is https://github.com/phetsims/joist/issues/657.

**Comment by pixelzoom** [comment]
To disable for the 10/1 deliverable, add to package.json:

```js
"phet": {
  "supportsZoom": false
```

**Comment by jessegreenberg** [comment]
Correct, that is the way to opt out (followed by `grunt update` to apply the change to the sim). I played with the sim with zoom for a while and it seems to be behaving well. Let me know if there's anything else I can do here!

**Comment by pixelzoom** [comment]
In Slack, @jessegreenberg mentioned that zoom was causing problems in a few sims, possibly related to WebGL.  Natural Selection uses WebGL (see `OrganismSprites`).

Relevant Slack messages in general channel:

Jesse Greenberg  5:35 PM
Hello all. The zoom feature is now enabled in all PhET sims on master with the exceptions of rutherford-scattering, density, and buoyancy. These were the only sims where I noticed substantial problems.  ...

Jesse Greenberg  5:55 PM
I am seeing a similar issue from density/bouyancy in charges-and-fields so it looks like the issue might be caused by zoom with the use of WebGL. Ill take a look at that soon.

Chris Malley  8:46 PM
Hmmm. Natural Selection uses WebGL, so be on the lookout there too.

**Comment by pixelzoom** [comment]
@jessegreenberg What is the timeframe for investigating the problem with WebGL sims? Do you want to do that in this issue, or create a new issue? 

FYI, Natural Selection uses WebGL. On 9/17, we start QA on a dev version that will be delivered to the client. Zoom can be disabled for the 9/17 deliverable. But then we immediate begin QA for production versions, and I believe this needs to be resolved for that process.

**Comment by pixelzoom** [comment]
In https://github.com/phetsims/joist/issues/657#issuecomment-689048282, @jessegreenberg noted that Zoom is causing problems with WebGL sims.  Natural Selection uses WebGL.   So Zoom may need to be disabled for the 10/1 deliverable, which begins dev testing on 9/17.  Unclear whether it would be blocking for production.

**Comment by pixelzoom** [comment]
Labeling blocks-sim-publication to remind me that Zoom needs to be either disabled or addressed for 10/1 deliverable.

**Comment by pixelzoom** [comment]
Over in https://github.com/phetsims/joist/issues/657#issuecomment-689800331, @ariel-phet said:

> @pixelzoom we should go with zoom enabled at first, but if we run into any significant issues, zoom can be disabled. Zoom does not _need_ to be enabled for the production version. Although desired, it is not a requirement. Moving forward we would certainly like to have zoom available in as many production sims as possible, but again, not a requirement for the coming production release.

@amanda-phet Note that this is a departure from the original plan, which was to publish the 10/1 dev version with Zoom disabled.  If you have anything related in the Client Guide, you may need to revise.

**Comment by pixelzoom** [comment]
I discussed this more with @ariel-phet on Slack.  Summary:

* Zoom + PhET-iO has not been tested. E.g. how does Zoom work in Studio? @ariel-phet or @jessegreenberg will create an issue to test compatibility and resolve issues.

* We will disable Zoom for the 10/1 deliverable.

* @amanda-phet will need to update the Client Guide for the 10/1 deliverable to indicate that `supportsZoom` defaults to `false`.

* If Zoom + PhET-iO testing hasn't been completed, or problems remain, Zoom will remain disabled for next production QA process.

**Comment by pixelzoom** [comment]
Zoom was disabled in the above commit.

**Comment by pixelzoom** [comment]
Note that in phetsims/chipper#980, `supportsZoom` was changed to `supportsPanAndZoom`.

**Comment by pixelzoom** [comment]
@ariel-phet asked me to enabled supportsPanAndZoom for the 1.3 release.

**Comment by pixelzoom** [comment]
supportsPanAndZoom has been enabled.  Ready for testing in next RC.

**Comment by KatieWoe** [comment]
Pan and zoom is implemented, but some issues may be present such as https://github.com/phetsims/natural-selection/issues/264. Leaving open due to this. @pixelzoom feel free to close if you think these aren't connected.

**Comment by pixelzoom** [comment]
I don't know if they are connected, and I'm not sure how to proceed.  I'll start by assigning to @jessegreenberg (pan and zoom) and the PhET-iO team (@zepumph and @samreid).   Please recommend whether this issue can be closed, or whether there is more to do here in order to be certain that pan and zoom is working correctly.

**Comment by jessegreenberg** [comment]
It looks like #264 was resolved today, so it should be fixed when you are ready to take SHAs for the RC.

> some issues may be present

@KatieWoe did you notice other problems besides #264 or was that the only one?

**Comment by KatieWoe** [comment]
#264 also impacted the recording wrapper. I also didn't get much of a chance to explore other wrappers, such as state, but that was the only one I'd seen so far

**Comment by pixelzoom** [comment]
I guess it's still in my court to make a decision, even though I know almost nothing about pan and zoom, or what was tested.  So based on comments by @jessegreenberg and @KatieWoe above, I'm going to infer that there's nothing else to do here, and this issue can be closed.   If that's not the case, please reopen ASAP.

**Comment by jessegreenberg** [comment]
Sounds good. If that was the only issue there is nothing else we are aware of and this can be closed.
